.github/workflows/dotnet-core.yml

```diff
@@ -35,8 +35,6 @@ jobs:
         path: '**/bin/**/*.*nupkg'
 
     - name: Publish MyGet
-      env:
-        MYGET_TOKEN: ${{ secrets.MYGET_TOKEN }}
       run: |
-        nuget setApiKey $MYGET_TOKEN  
+        nuget setApiKey ${{ secrets.MYGET_TOKEN }}  
```

Comment:
I have slightly different opinion on how we want to manage the configuration/secrets:
1. We probably want the action name to describe the intention, for example "Publish NuGet Packages".
2. The configuration probably needs to be agnostic to the target, for example `NUGET_PACKAGE_SOURCE`, `NUGET_API_KEY`.